python/rola/core.py

```diff
@@ -0,0 +1,52 @@
+from radix_engine_toolkit import derive_virtual_account_address_from_public_key, PublicKey
+
+from rola.models.signed_challenge import SignedChallenge
+from rola.utils.gateway import get_entity_owner
+from rola.utils.helpers import create_public_key_hash
+from rola.utils.ret import derive_address
+
+
+class Rola:
+
+    def __init__(self,
```

Comment:
Can we use a formatter? I would recommend using [black](https://github.com/psf/black).